FuncGameDev/FSharp/Unity/Update.fs

```diff
@@ -15,6 +15,7 @@ type Updater() =
         ()
 
     member this.Update() =
+        CameraInterpreter.updateCamera
```

Comment:
Why is this here? This is literally just a function, right? You're not even calling it, just stating it?